frontend/src/features/properties/components/forms/subforms/InformationForm.tsx

```diff
@@ -19,7 +19,7 @@ export const defaultInformationFormValues = {
   landLegalDescription: '',
   classificationId: '',
   agencyId: '',
-  isSensitive: false,
+  isSensitive: '',
```

Comment:
Why is we always use empty string instead of `undefined` or `null`?

Reply:
Formik requires that all fields are initialized with a non-null, non-undefined value or you will get a warning about changing an uncontrolled input to a controlled one when you enter data into that field at runtime.

so generally, all formik forms should be provided with an initialValues object where all fields of the form are explicitly given a value. in most cases the typings for those values should be:
someField: theActualType | '';